superset/db_engine_specs/presto.py

```diff
@@ -423,24 +423,27 @@ def get_function_names(cls, database: Database) -> List[str]:
         return database.get_df("SHOW FUNCTIONS")["Function"].tolist()
 
     @classmethod
-    def _partition_query(  # pylint: disable=too-many-arguments,too-many-locals
+    def _partition_query(  # pylint: disable=too-many-arguments,too-many-locals,unused-argument
         cls,
+        schema: Optional[str],
```

Comment:
Given this is a private method, changing the order of the function arguments is deemed non-breaking.